tests/table.html

```diff
@@ -0,0 +1,9 @@
+<table>
+<tr><th>1st field </th><th>2nd field </th><th>3rd field </th></tr>
+<tr><td class="left">1st entry </td><td class="center">2nd entry </td><td class="right">3rd entry </td></tr>
```

Comment:
Also most other converters use `align="${alignment}"` or `style="text-align: ${alignment};"` , probably should follow either of these two instead of doing something different than most other parsers.

Reply:
Fixed in https://github.com/karlb/smu/pull/11/commits/889ca7b25da7e78b7aba7b9c3006bb0f9df59ce6

Reply:
The `align` attribute is obsolete and not valid HTML5, see https://html.spec.whatwg.org/multipage/obsolete.html#non-conforming-features. The modern replacement would be `style="text-align: ${alignment};"`, which makes sense because it will cause alignment without requiring any CSS rules (like the `class` approach does).

Having the classes might still be useful to allow users styling based on the chosen alignment (e.g. more left padding for left aligned columns). I'm not sure that case is frequent enough to warrant adding the class markup on top of the `text-align`, though.

Reply:
> The `align` attribute is obsolete and not valid HTML5

Hmm, wasn't aware of that. Saw other parser use it and picked it as it was the shorter of the two. We can just switch it to `style=text-align..` in that case.